samples/DataSourceWithProcessingAndExtensions/DataGenerator/Jsonbinary.cs

```diff
@@ -0,0 +1,41 @@
+using System.IO;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace DataGenerator
+{
+    public static class JsonBinary
+    {
+        public static byte[] ToBinary(object value)
+        {
+            byte[] result = null;
+
+            using (var stream = new MemoryStream())
+            {
+                using (var writer = new BinaryWriter(stream, Encoding.Default, true))
```

Comment:
On .NET Core, this is fine as it is *always* UTF8. However, given that this is reference code and other versions of the framework do not make guarantees about what the encoding is, and the encoding can change on a single computer, we should be more explicit about what to use (i.e. Encoding.UTF8).

https://docs.microsoft.com/en-us/dotnet/api/system.text.encoding.default?view=netcore-3.1